package.json

```diff
@@ -51,8 +51,7 @@
     "ansi-regex": "^5.0.1",
     "set-value": "^4.0.1",
     "minimist": "1.2.8",
-    "semver": "^7.5.3",
-    "word-wrap": "npm:@aashutoshrathi/word-wrap"
```

Comment:
This is unrelated, just a small fix I wanted to do anyway (the regular word-wrap has security vulnerabilities fixes in now so we don't need to rely on this version anymore)